pages/user/[id].js

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from 'react'
+import { useRouter } from 'next/router'
+import Results from '../../components/results'
+import useStore from '../../components/store'
+
+const User = () => {
+  const router = useRouter()
+  const id = router.query.id
+
+  const search = useStore((state) => state.search)
+  const searchBy = useStore((state) => state.searchBy)
+  const setSearchBy = useStore((state) => state.setSearchBy)
+  const setShowResultsBy = useStore((state) => state.setShowResultsBy)
+  const setSearch = useStore((state) => state.setSearch)
+  const searchId = useStore((state) => state.searchId)
+
+  useEffect(() => {
+    router.beforePopState(({ as }) => {
+      if (as !== router.asPath && as === '/user') {
+        setSearch('')
+      }
+      return true
+    })
+
+    return () => {
+      router.beforePopState(() => true)
+    }
+  }, [router])
+
+  useEffect(() => {
+    if (!searchBy.user) {
+      setSearchBy({ project: false, user: true, facility: false })
```

Comment:
nit: these repeated `setSearchBy` and `setShowResultsBy` calls in each `/[object]` `/[object]/[id]` pair of routes make me wonder if actions like `setUserDefaults()` would be helpful. Maybe not so important since we're unlikely to ever need to change the defaults.

Reply:
no longer using this bit, but, did a similar refactoring for another bit, good suggestion!